lms/templates/api_admin/catalogs/list.html

```diff
@@ -26,6 +26,7 @@ <h1 id="api-header">${_("Catalogs for {username}").format(username=username)}</h
     % for catalog in catalogs:
     <li>
       <a href="${reverse('api_admin:catalog-edit', args=(catalog.id,))}">${catalog.name}</a>
+      <a href="${reverse('api_admin:catalog-csv', args=(catalog.id,))}">download CSV</a>
     </li>
```

Comment:
Have you got a screenshot? I'm not sure why there are multiple links within a list item.